mirrord-agent/src/sniffer.rs

```diff
@@ -93,12 +93,73 @@ fn is_closed_connection(flags: u16) -> bool {
     0 != (flags & (TcpFlags::FIN | TcpFlags::RST))
 }
 
-struct ConnectionManager {
-    sessions: SessionMap,
-    index_allocator: IndexAllocator<ConnectionID>,
-    ports: HashSet<u16>,
+pub struct TcpManagerCodec {}
```

Comment:
Empty struct could be turned into just a marker struct (also add a bunch of derives to it if it's going to be just a marker).

Reply:
Can you elaborate? I'm not sure I follow

Reply:
Just saying that if this struct has no fields, drop the `{}` (and feel free to add derive Clone, Debug, and so on).